float32 meters_back
---
geometry_msgs/PoseStamped previous_pose
